test/functional/interface_bitcoin_cli.py

```diff
@@ -3,20 +3,33 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test bitcoin-cli"""
+
+from decimal import Decimal
+
 from test_framework.test_framework import BitcoinTestFramework
-from test_framework.util import assert_equal, assert_raises_process_error, get_auth_cookie
+from test_framework.util import (
+    assert_equal,
+    assert_raises_process_error,
+    connect_nodes,
+    get_auth_cookie,
+)
+
+BLOCKS = 101
+BALANCE = Decimal('50.00000000')
```

Comment:
nit: Maybe a comment is warranted how come after generating 101 blocks the node's balance becomes 50.

```suggestion
# The block reward is 50 but it only matures after 100 (COINBASE_MATURITY) blocks,
# so the node "gets" the reward for the first block as trusted 100 blocks later.
BLOCKS = 101
BALANCE = Decimal('50.00000000')
```

Reply:
Done. This seems to be a standard convention in the tests, explained in test-shell.md, and used without documentation in wallet-basic.py and rpc_rawtransaction.py, among others... but I added a doc since it's generally better to do so.